client/src/redux/failed-updates-epic.js

```diff
@@ -45,7 +50,15 @@ function failedUpdateEpic(action$, state$) {
     filter(() => store.get(key)),
     filter(() => isOnlineSelector(state$.value)),
     tap(() => {
-      const failures = store.get(key) || [];
+      let failures = store.get(key) || [];
+
+      // delete unsubmitable failed challenges
+      if (!failures.every(isSubmitable)) {
+        let submitableFailures = failures.filter(isSubmitable);
```

Comment:
Can be removed if you commit the above change.
```suggestion
```

Reply:
GitHub now supports multiple line comments in the save review comment: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request :)

Reply:
I was trying to find how to do that, but I thought it was not implemented yet.  Thanks for the information.